### PR TITLE
Add password changed email notification

### LINKED ISSUE #2439: Add Password Changed Email Notification
**What type(s) of user does this feature affect?**
 - volunteers
 - supervisors
 - admins

**Description**
Currently when a user changes their password the only response is a banner saying: "Password was successfully updated". 
We would like an email to be sent to users alerting them that their password has been changed as well. This email should follow the same general template as other CASA emails and have a subject line of "CASA Password Changed" and the texts of the email should read:

"Hello (username), 

Your CASA password has been changed. 

If you have any questions, please contact a (Name of relevant CASA) CASA administrator for assistance.
".

**Screenshots of current behavior, if any**  
Password Updated

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 123456

### PR DESCRIPTION
**Pull request by mauricio-prl**
### What github issue is this PR for, if any?
Resolves #2439

### What changed, and why?

To ensure users will know if their password has changed.

### How is this tested? (please write tests!) 💖💪

Unit tests for the mailer, ensuring it will be called in controllers and with capybara.

### CODE REVIEW

File: app/controllers/all_casa_admins_controller.rb [hunk 1]
```diff
@@ -25,7 +26,7 @@ def update
     @user = current_all_casa_admin
 
     if @user.update(all_casa_admin_params)
-      flash[:success] = "Profile was successfully updated."
+      flash[:success] = 'Profile was successfully updated.'
```
Comment (josearmandojacq) [inline app/controllers/all_casa_admins_controller.rb:29]: Hey, nice work. Can you please use this part of the README 🙂: 

Cleaning up before you pull request

bundle exec standardrb --fix auto-fix Ruby linting issues more linter info
bundle exec erblint --lint-all --autocorrect ERB linter
yarn lint:fix to run the JS linter and fix isses
rake factory_bot:lint if you have been editing factories and want to find factories and traits which produce invalid objects

This way we all have an unified code base.

**Review by compwron — COMMENTED** [review]
This is looking promising! Please add another test with capybara to make sure that the email is sent at the proper time when the button is clicked on the UI

**Comment by compwron** [comment]
We use standardrb which is a specific set of rubocop configurations :)

File: app/controllers/all_casa_admins_controller.rb (hunk 1 above)
Comment (m-pereira) [reply app/controllers/all_casa_admins_controller.rb]: Thank so much! I did realize that.

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class UserMailer < ApplicationMailer
+  def password_changed_reminder(user)
+    @user = user
+    @casa_organization = user.try(:casa_org) || nil
```
Comment (compwron) [inline app/mailers/user_mailer.rb:4]: fair enough

**Review by compwron — APPROVED** [review]
Yay tests! :)

**Pull request merged**